test/hsm/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM letsencrypt/boulder-tools:2016-07-05
+MAINTAINER Roland Bracewell Shoemaker <roland@letsencrypt.org>
+
+COPY test-ca.key /root/key.pem
+
+RUN echo "0:/var/lib/softhsm/slot0.db" > /etc/softhsm/softhsm.conf && \
+    softhsm --init-token --slot 0 --label key --pin 1234 --so-pin 0000 && \
```

Comment:
Actually, why not put these key installation steps in the main boulder-tools Dockerfile and run it with a different command? That would mean one less entity to think about.